src/content/reference/platform-api/crd/certconfigs.core.giantswarm.io.md

```diff
@@ -0,0 +1,398 @@
+---
```

Comment:
Not relevant for CAPI. We don't install cert-operator.

Reply:
I removed awsclusters.infrastructure.giantswarm.io. Will need your help to detect other redundant ones.